beaker.gemspec

```diff
@@ -53,6 +53,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'stringify-hash', '~> 0.0'
   s.add_runtime_dependency 'beaker-hiera', '~> 0.0'
   s.add_runtime_dependency 'beaker-hostgenerator'
+  s.add_runtime_dependency 'thor'
```

Comment:
@ferglor, have you or @tvpartytonight done any experiments or reading on any other options for implementing subcommands? One potential contender @nwolfe brought up was [commander](https://github.com/commander-rb/commander), have we looked at this?

@rick have any advice for implementing subcommands in the ruby world?